src/trajectory_loader_node.cpp

```diff
@@ -64,7 +64,7 @@ Trajectory TrajectoryLoaderNode::createTrajectory(const Points & points)
   for (auto & point : points) {
     TrajectoryPoint trajectory_point;
     Pose pose;
-    auto q = tier4_autoware_utils::createQuaternionFromYaw(point[2]);
+    auto q =  autoware_universe_utils::createQuaternionFromYaw(point[2]);
```

Comment:
```suggestion
    auto q =  autoware::universe_utils::createQuaternionFromYaw(point[2]);
```
thank you for using autoware and sorry for inconvenience. we changed also name values
https://github.com/autowarefoundation/autoware.universe/pull/7594

Reply:
Thank you. The namespace was recently changed. I have made modifications.